docs/4_connectivity.md

```diff
@@ -253,7 +255,7 @@ int main(int, char**) {
     pirCount->methods(M2MMethod::GET);
     pirCount->observable(true);
 
-    printf("Connecting to Mbed Cloud...\n");
+    printf("Connecting to Device Management...\n");
```

Comment:
Same here as everywhere else, it's called Pelion Device Management as far as I know.

Reply:
Only top-level first mentions of it